### ISSUE TITLE
Options.FullTripleIndexing affects number of SPARQL query results

### ISSUE THREAD

**Issue by bent-rasmussen** [dotnetrdf/dotnetrdf#466]
Is it intended that `Options.FullTripleIndexing` should affect the number of triple results?

**Example graph**

```rdf
@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
@prefix : <urn:edge/>.
@prefix al: <urn:al/>.

_:autos1005 a al:Foo;
            :name "bla".
_:autos1004 a al:Bar;
            :name "bla".
```

**Example query**

```rdf
PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
PREFIX al: <urn:al/>
PREFIX : <urn:edge/>

SELECT * WHERE { ?p a al:Bar; :name 'bla' }
```

**Results**

`Options.FullTripleIndexing = true` gives 1 result.
`Options.FullTripleIndexing = false` gives 2 results.

Setting it to false seems to imply that querying doesn't make much sense. Is that correctly understood or is this an implementation bug?

**Comment by kal** [comment]
Thanks for the report. This does sound like a bug - disabling full triple indexing shouldn't modify the results you get, just how quickly you get them.

**Comment by kal** [comment]
Fixed in dotNetRDF 2.7.3 which is just released to NuGet.
